release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -156,6 +156,7 @@ With this release, there are several updates to the *Administrator* perspective
 
 * The {product-title} web console displays a `ConsoleNotification` if the cluster is upgrading. Once the upgrade is done, the notification is removed.
 * A *_restart rollout_* option for the `Deployment` resource and a *_retry rollouts_* option for the `DeploymentConfig` resource are available on the *Action* and *Kebab* menus.
+* You can view a list of supported cluster products on the *_Cluster type_* dropdown menu on the the *{product-title} Cluster Manager* page. The supported products include {product-title}, {product-title} (ROSA), Azure Red Hat OpenShift (ARO), and {product-dedicated}. 
```

Comment:
Not sure if you mean the Cluster List dropdown in OCP console, or some particular dropdown in ACM Console?
```suggestion
* You can view a list of supported cluster products on the *_Cluster list_* dropdown menu on the the *{product-title} Cluster Manager* page. The supported products include {product-title}, {product-title} (ROSA), Azure Red Hat OpenShift (ARO), and {product-dedicated}. 
```

Also allowed cluster types are: OpenShift, ROSA, ARO, ROKS, and  OpenShiftDedicated
ROKS is missing in the docs

Reply:
So we should add something about the allowed cluster types to https://docs.openshift.com/container-platform/4.11/web_console/web-console.html#multi-cluster-about_web-console?